app/routes/session.py

```diff
@@ -127,11 +127,14 @@ def get_sign_out():
     Signs the user first out of eq, then the account service by hitting the account services'
     logout url.
     """
+    redirect_url = request.args.get("redirect_url") or cookie_session.get(
+        "account_service_log_out_url"
+    )
+
     logout_user()
 
-    account_service_log_out_url = cookie_session.get("account_service_log_out_url")
-    if account_service_log_out_url:
-        return redirect(account_service_log_out_url)
+    if redirect_url:
```

Comment:
Petty comment but can the `redirect_url` var be moved to after `logout_user()` is called?

Reply:
Since you asked so nicely 👍